paper/paper.md

```diff
@@ -86,6 +86,18 @@ where:
 - $\omega$ is the rotor whirl speed;
 - $\mathbf{f}$ is the generalized force vector.
 
+ROSS is capable of evaluating the impact that components such as bearings and seals have on the rotor.
+The numerical models for calculating the dynamic coefficients have relatively high error comparing to experimental
+data [@jolly2018]. Particularly, ROSS' differential is its capability to perform analysis on how the uncertainties
+on these coefficients can impact the rotor dynamic. Thus, through the stochastic analysis performed on ROSS the user
+is able to obtain the probability of failure of a rotor.
+
+Other parallel projects are intended to work with ROSS, generating data to test learning algorithms
+that should diagnose machine failure. ROSS generates a rotor with some problem such as an
+unbalance or misalignment, and then the output data from this model is used to test the learning algorithm
+and see how good it is in diagnosing the problem; ROSS is also interesting for model updating.
+Here we could use vibrations measurements from a rotor to update a model generated by ROSS.
```

Comment:
Change to:
>ROSS has been used to evaluate the impact of damper seal coefficients uncertainties in a compressor rotordynamics [@timbo2019]. 
Other projects are also using ROSS to test machine learning algorithms that can diagnose machine failure. Here we can create a model with some problem such as unbalance or misalignment and then use the output data from this model to test the learning algorithm and see how good it is in diagnosing the problem.